clients/retrieval_client.go

```diff
@@ -157,5 +179,12 @@ func (r *retrievalClient) RetrieveBlob(
 		return nil, err
 	}
 
+	// Log Operator Fetch Chunk Status
+	for _, statusInfo := range operatorFetchChunkStatus {
+		if !statusInfo.Success {
+			r.logger.Warn("Failed to receive chunks from operator", statusInfo.IndexedOperatorInfo)
```

Comment:
The logger has the following signature: 
`Warn(message, key1, value1, key2, value2,...)`,

so you are missing a key.